applications/nrf_desktop/src/modules/hid_state.c

```diff
@@ -538,7 +543,8 @@ static void send_report_keyboard(void)
 		     i++) {
 			struct item item = rd->items.item[max - i - 1];
 
-			if (item.value) {
+			if (item.usage_id) {
+				__ASSERT_NO_MSG(item.value == 1);
```

Comment:
this is incorrect - value can be higher then one if we have more then one physical key assigned to the same usage

Reply:
`assert(item.value > 0)` instead

Reply:
done